<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>332</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>語音辨識測試</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>250</width>
      <height>220</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>250</width>
      <height>220</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>你說的</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>要比對的</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>開始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="compareButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>相似度</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>168</y>
       <width>47</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>SimHash</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>208</y>
       <width>47</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Liklihood</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>plainTextEdit_2</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>關於</string>
   </property>
   <property name="iconText">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>離開</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
